indexing-service/src/main/java/org/apache/druid/indexing/common/task/batch/parallel/ParallelIndexSupervisorTask.java

```diff
@@ -545,16 +811,35 @@ public Response getMode(@Context final HttpServletRequest req)
     return Response.ok(isParallelMode() ? "parallel" : "sequential").build();
   }
 
+  @GET
+  @Path("/phase")
+  @Produces(MediaType.APPLICATION_ATOM_XML)
```

Comment:
Why is `MediaType.APPLICATION_ATOM_XML` used here?

Reply:
Oops, should be `MediaType.APPLICATION_JSON`. Fixed.